packages/fgtclb/academic-persons/ext_localconf.php

```diff
@@ -82,6 +83,15 @@
         ExtensionUtility::PLUGIN_TYPE_CONTENT_ELEMENT
     );
 
+    ExtensionManagementUtility::addTypoScriptSetup(trim('
```

Comment:
This should not be added as overall system global, which does not make much sense.

Better move that to the corresponding TypoScript files already loaded / selected using SiteSets (if done):

* Configuration/TypoScript/Default/setup.typoscript
* Configuration/TypoScript/Standalone/setup.typoscript

Global loaded TypoScript can eventually vanish depending on `sys_template` option selection and kind of fragile.

The title provider does not be registered globally, specially if content / plugin is not used within a specific site - and the selective files is a more conveniant place to do so.